### DISCUSSION TITLE
Codebase as context

### DISCUSSION THREAD

**Discussion by srikat** [zed-industries/zed#10830] [Feature Requests]
Hi. Is there a way to make AI in Zed (after inputting my OpenAPI key) use all the files in the current project i.e, the current codebase as the context when it answers questions?

![CleanShot 2024-04-22 at 12  16 19@2x](https://github.com/zed-industries/zed/assets/2188656/b87c79cc-ab22-4936-91fe-8117e9bf0ddd)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
